docs/data-management/metrics/certified-metrics.md

```diff
@@ -29,27 +29,181 @@ metrics:
     operation: sum
 ```
 
-Metrics are certified via the [metric sync API](https://eppo.cloud/api/docs#/Metrics%20Sync/syncMetrics). You can either post to that endpoint directly or, if using GitHub, add a simple GitHub action using the `eppo-metrics-sync` python package (see below for details). 
+Metrics are certified via the [metric sync API](https://eppo.cloud/api/docs#/Metrics%20Sync/syncMetrics). You can either post to that endpoint directly or use the `eppo_metrics_sync` [python package](https://github.com/Eppo-exp/eppo-metrics-sync) to post a directory of yaml files to Eppo's API. See [Syncing Metrics](#syncing-metrics) below for details. 
 
-Any metric or fact created through the metric sync API will have a Certified badge and will not be editable in the UI:
+Any metric or fact source created through the metric sync API will have a Certified badge and will not be editable in the UI:
 
 ![Certified metric example](/img/metrics/certified-metrics-1.png)
 
-## Certified Metric Schema
-Eppo supports a YAML schema to describe Facts and Metrics.
+## Certified Metric schema
+Eppo supports a yaml schema to describe Facts and Metrics. Each yaml file can define fact sources, metrics, or both. This section describes the yaml schema in detail and provides a few [examples](#examples).
 
-The Eppo certified metrics schema is [described here](https://eppo.cloud/api/docs#/Metrics%20Sync/syncMetrics)
+### Fact sources 
 
-Notes:
-- Metrics are unique based on the `name` field.
-- `sync_tag` describes the source of the synced metrics and will show in the UI
-- The following fields are optional: descriptions, reference URLs, properties, filters, and winsorization percentiles
+Certified metric files can define one or more [fact sources](/data-management/definitions/fact-sql). The fact source schema is shown below:
 
-## Sync with a Github repository
+| Property |  Description | Example |
+| -------- | ----------- | ------- | 
+| `name` | A name for the source to display in the Eppo UI | Purchases |
+| `sql` | The SQL definition for the source | <pre><code>select * <br></br>  from analytics_prod.purchases </code></pre> |
+| `timestamp_column` | The column that represents the time the fact occurred | `purchase_timestamp` |
+| `entities` | A list of [entities](/data-management/entities) in the table, each element containing the name of an existing entity created in the Eppo UI and the corresponding column | <pre><code>- entity_name: User <br></br>  column: user_id </code></pre> |
```

Comment:
```suggestion
| `entities` | A list of [entities](/data-management/entities) in the table, each element containing the name of an existing entity created in the Eppo UI and the corresponding column | <pre><code>entity_name: User <br></br>  column: user_id </code></pre> |
```

Reply:
Those dashes are intentional to indicate that `entities` is a list (you'd copy that example for each element in the list, and specify new entries with a `-`)